openmetadata-ui/src/main/resources/ui/src/pages/TagsPage/TagsPage.tsx

```diff
@@ -434,6 +434,7 @@ const TagsPage = () => {
       const patchData = compare(editTag, updatedData);
       try {
         await patchTag(editTag.id ?? '', patchData);
+        classificationDetailsRef.current?.refreshClassificationTags();
```

Comment:
Why we want to refresh the tags on patch req?

Reply:
The issue is the fetch request is written in the child component i.e the ClassificationDetails and the patch request is made in parent component ie the TagsPage. the list is updated in the tagspage and not in the ClassificationDetails so implemented that logic